app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -67,39 +60,35 @@ If your application is successful, the training provider may be able to help you
+Most schools in England need you to have [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach. Without QTS you may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.
 
 Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 
 
-Without QTS, you will not eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.
+If your teacher training course leads to QTS, you may be eligible for a scholarship or bursary to help you train.
 
 ## If the course is fee-funded or salaried
 
 ### Fee-funded teacher training
 
-Most teacher training courses are fee-funded – this means you have to pay tuition fees and will not earn a salary while you train. 
+Most teacher training courses are fee-funded. This means you have to pay tuition fees and will not earn a salary while you train. 
 
 There are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship.
 
 You can [find out about the different ways to fund your training](/funding-and-support).
 
 <div class="inset">
 
-<p> If you’re a non-UK citizen without indefinite leave to remain in the UK, you will not usually be eligible for a bursary or scholarship, unless you train to teach languages or physics.</p>
+<p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. </p> 
+
+<p> Other non-UK citizens will not be eligible for student finance, but may be eligible for a bursary or scholarship to train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>
 
 </div>
 
 ### Salaried teacher training
 
 Some courses are salaried, which means you’ll receive an unqualified teacher’s salary while you train. 
 
-Places on salaried courses are limited and very competitive. You can [find out what you’ll need to get on a salaried teacher training course](/funding-and-support/salaried-teacher-training). 
-
-<div class="inset">
-
-<p>If you’re a non-UK citizen, the chance of getting a place on a salaried course is very limited.</p>
-
-</div>
+Places on salaried courses are limited and very competitive. If you’re a non-UK citizen, you're very unlikely to get a place on a salaried course. You can [find out what you’ll need to get on a salaried teacher training course](/funding-and-support/salaried-teacher-training). 
```

Comment:
Is it worth putting the link on a separate line - is there any danger that UK citizens might not realise that link is for them too as it comes straight after the non-UK sentence?

(But I agree that moving the non-UK sentence up makes sense here)